src/styles/blocks/header.scss

```diff
@@ -0,0 +1,72 @@
+.header {
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  background-color: white;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  color: $main-accent-color;
+
+  &__logo {
+    display: block;
+    width: 40px;
+    height: 40px;
+    margin: 10px 0;
+  }
+}
+
+.nav {
```

Comment:
This is separate block, move it to separate file

Reply:
Pay attention to comments.